### PR TITLE
chore: dl-card component and story LESQ-8

### PR DESCRIPTION
**Pull request by k-huggs**
## Description

- Creates DLCard slim download component in storybook
- Add grey60 color
- Updates Checkbox/Visual Checkbox components to 

## AC:

- [x]  Uses a new slimline card design
- [x]  Shows the format of the file
- [x]  Has clickable tick box
- [x]  Has different states (see Figma)

## Issue(s)

Fixes #LESQ-8

## How to test

1. Go to https://deploy-preview-2045--owa-storybook.netlify.thenational.academy/?path=/docs/download-components-dl-card--docs
2. Click on DLCard component on 
3. You should see slimline DL card
4. Toggle error to see error state
5. Click to see focus state

## Screenshots

How it should now look:

Screenshot 2023-10-25 at 09 28 32
Screenshot 2023-10-25 at 09 28 55

## Checklist

- [x] Added or updated tests where appropriate
- [x] Manually tested across browsers / devices
- [x] Considered impact on accessibility
- [x] Design sign-off
- [x] Approved by product owner
- [x] Does this PR update a package with a breaking change

### CODE REVIEW

File: src/components/DownloadComponents/DLCard/DLCard.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+};
+
+type DownloadCardLabelProps = DownloadCardProps & {
+  isHovered: boolean;
+};
+
+const BoxWithFocusState = styled.div`
+  position: relative;
+  border: solid 4px transparent;
+  /* cursor: pointer; */
```
Comment (njblore) [inline src/components/DownloadComponents/DLCard/DLCard.tsx:30]: 👁️

File: src/components/DownloadComponents/DLCard/DLCard.tsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+  const { hoverProps, isHovered } = useHover({});
+
+  return (
+    <>
+      {hasError && (
+        <Flex>
+          <Icon name="content-guidance" $color={"teachersRed"} $width={24} />
+          <P $color={"teachersRed"} $font={"body-2"}>
+            error message
```
Comment (njblore) [inline src/components/DownloadComponents/DLCard/DLCard.tsx:107]: should this be the actual error? it also seems to be sitting within the card at the mo 
Screenshot 2023-10-25 at 13 35 45

**Review by njblore — CHANGES_REQUESTED** [review]
looks good just a couple of comments!

**Review by Monika-Koziol — CHANGES_REQUESTED** [review]
@k-huggs  looking good a few little tweaks
- [x] Hover state: the white should change to grey 20 f2f2f2
- [x]  The second line of text: currently it forces sentence case. Can it be just any input? We might sometimes have all caps in there.
- [ ] If someone enters a very long text could it truncate in the top line? 
More of the question but how are the icons managed? I wasn’t able to change them

**Comment by k-huggs** [comment]
> @k-huggs looking good a few little tweaks
> 
> * [x]  Hover state: the white should change to grey 20 f2f2f2
> * [ ]   The second line of text: currently it forces sentence case. Can it be just any input? We might sometimes have all caps in there.
> * [ ]  If someone enters a very long text could it truncate in the top line?
>   More of the question but how are the icons managed? I wasn’t able to change them

1. Changed the hover state ✅
2. Think it would be better to keep them all the same either capital case or capitalised
3. The longest text possible is 'Additional material' for these cards as they need to be a selected from a resource type

Try the following to change images in the resource type
"intro-quiz-questions", "intro-quiz-answers", "exit-quiz-questions", "exit-quiz-answers", "worksheet-pdf", "worksheet-pptx", "supplementary-pdf", "supplementary-docx"

**Comment by Monika-Koziol** [comment]
@k-huggs looking good from my side.

**Review by njblore — APPROVED** [review]

**Review by Monika-Koziol — APPROVED** [review]

**Pull request merged**